cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/goals/LeaderBytesInDistributionGoal.java

```diff
@@ -262,9 +262,10 @@ public int compare(ClusterModelStats stats1, ClusterModelStats stats2) {
         return 1;
       }
 
-      double[] stat2 = stats2.utilizationMatrix()[RawAndDerivedResource.LEADER_NW_IN.ordinal()];
-      double variance1 = new Variance().evaluate(stat1);
-      double variance2 = new Variance().evaluate(stat2);
+      // If there are brokers with inbound network load over the threshold, the standard deviation of utilization
```

Comment:
Since we are dropping `utilizationMatrix`, does it make sense to use it [here?](https://github.com/linkedin/cruise-control/pull/298/files#diff-1244a6c28981381d0bd3667ff5749a6fR258)